backend/api/Controllers/MissionController.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Text.Json;
+using Api.Models;
+using Api.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Api.Controllers;
+
+[ApiController]
+[Route("missions")]
+public class MissionController : ControllerBase
+{
+    private readonly ILogger<MissionController> _logger;
+
+    private readonly EchoService _echoService;
+
+    public MissionController(ILogger<MissionController> logger, EchoService echoService)
+    {
+        _logger = logger;
+        _echoService = echoService;
+    }
+
+    /// <summary>
+    /// List all available missions on the asset
+    /// </summary>
+    /// <remarks>
+    /// Overview
+    ///
+    /// List all available missions on the asset in the Echo mission planner
+    /// </remarks>
+    [HttpGet]
+    [ProducesResponseType(typeof(List<Mission>), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    public async Task<ActionResult<IList<Mission>>> GetMissions()
+    {
+        try
+        {
+            var missions = await _echoService.GetMissions();
+
+            return Ok(missions);
+        }
+        catch (Exception e) when (e is MissionNotFoundException || e is JsonException)
+        {
+            _logger.LogError(e, "Missions not found");
+
```

Comment:
```suggestion
```

Just because I think c# produces enough lines and I think it looks just as clean without a whiteline here. Open for discussion.